bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/VisualizationTabPane.jsx

```diff
@@ -44,6 +45,9 @@ export const VisualizationTabPane = ({ layer, capabilities, scales, propertyFiel
             { propertyFields.includes(STYLE) &&
                 <Style layer={layer} capabilities={capabilities} controller={controller} propertyFields={propertyFields} />
             }
+            { propertyFields.includes(STYLE) &&
```

Comment:
This only applies to styles defined as Oskari style JSON so:
```suggestion
            { propertyFields.includes(STYLES_JSON) &&
```